testnet/tools/nns-tools/add-release-to-changelog.sh

```diff
@@ -74,10 +74,10 @@ else
     print_red "(In particular, unable to determine which canister and commit.)" >&2
     exit 1
 fi
-SECONDS_AGO=$(($(date +%s) - "${EXECUTED_TIMESTAMP_SECONDS}"))
-EXECUTED_ON=$(
+SECONDS_AGO=$(($(date +%s) - "${PROPOSED_TIMESTAMP_SECONDS}"))
+PROPOSED_ON=$(
     date --utc \
-        --date=@"${EXECUTED_TIMESTAMP_SECONDS}" \
+        --date=@"${PROPOSED_TIMESTAMP_SECONDS}" \
         --iso-8601
 )
 print_cyan "🏃 ${GOVERNANCE_TYPE} ${CANISTER_NAME} proposal was executed ${SECONDS_AGO} seconds ago." >&2
```

Comment:
also here, this comment will no longer be correct (maybe do a sweep of the whole file)

Reply:
sweeped!